### PR TITLE
Fix to let branch can contain slash

### PR DESCRIPTION
**Pull request by adaam**
I meet a problem when I use this script to sync files with branch name with slash.
I think it does not need to split with slash when parsing branch.
But I'm not a javascript expert, just found it will work when branch name with slash.
Please help to review if it makes sense.

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -205,4 +205,4 @@ const parseConfig = async () => {
 module.exports = {
 	...context,
 	parseConfig
-}
\ No newline at end of file
+}
```
Comment (BetaHuhn) [inline src/config.js:208]: Please remove this newline at the end.

**Review by BetaHuhn — CHANGES_REQUESTED** [review]

**Comment by BetaHuhn** [comment]
Hey @adaam,

Thanks for taking the time to create this PR! Syncing a branch with a `/` is totally a valid use case which I oversaw when writing the code. 

If you could remove the added newline at the end, I will merge it!

File: src/config.js (hunk 1 above)
Comment (adaam) [reply src/config.js]: Sure. Newline removed.

**Comment by adaam** [comment]
Newline in two files are removed.

**Comment by BetaHuhn** [comment]
Thanks!

**Pull request merged**